rules/const.bzl

```diff
@@ -71,6 +71,29 @@ CONST = struct(
     ),
 )
 
+def get_lc_items(*want_lc_values):
+    """Return a list of key-value pairs from CONST.LCV with lowercased keys.
+
+    If `want_lc_values` is empty, returns an unfiltered dict. Otherwise, only
+    key-value pairs where the value is in `want_lc_values` are returned.
+    """
+    lcv_dict = structs.to_dict(CONST.LCV)
+
+    for want_val in want_lc_values:
+        if want_val not in lcv_dict.values():
+            fail("Unrecognized LCV: {}".format(want_val))
+
+    out = []
+    for key, val in lcv_dict.items():
+        key = key.lower()
+        if len(want_lc_values) == 0 or val in want_lc_values:
+            out.append((key, val))
+
+    if len(want_lc_values) != 0 and len(out) == 0:
+        fail("get_lc_dict would produce an empty list")
+
+    return out
+
```

Comment:
I'm curious what reviewers will think of this new function. It's a little additional magic, but I think it makes the Bazel test definitions more ergonomic. If this is well-received, I can go further and replace  a bunch of `structs.to_dict(CONST.LCV)` with `get_lc_items()` (and a bunch of `lc_state.lower()` with `lc_state`).

Reply:
SGTM but PTAL at my comment above.